common/shlibs

```diff
@@ -3976,3 +3976,4 @@ libptexenc.so.1 texlive-20200406_1
 libdolphinvcs.so.5 dolphin-20.04.3_1
 libcglm.so.0 cglm-0.7.6_1
 libfcft.so.3 fcft-2.2.2_1
+libtdjson.so.1.6.6 libtd-1.6.6_1
```

Comment:
You can put this in the middle of the file (e.g. close to related libs) to avoid conflicts all the time.